### PR TITLE
Reduce number of buckets in ttl controller for 2k+ nodes clusters

### PR DESCRIPTION
**Pull request by marseel**
#### What type of PR is this?
/kind bug

#### What this PR does / why we need it:
Currently, in clusters with ~10k nodes, when controller leader changes, current number of nodes might change slightly and ttl controller might need to process all nodes in cluster to update ttl causing increased load on apiserver.
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/controller/ttl/ttl_controller.go [hunk 1]
```diff
@@ -108,8 +108,7 @@ var (
 		{sizeMin: 90, sizeMax: 500, ttlSeconds: 15},
 		{sizeMin: 450, sizeMax: 1000, ttlSeconds: 30},
 		{sizeMin: 900, sizeMax: 2000, ttlSeconds: 60},
-		{sizeMin: 1800, sizeMax: 10000, ttlSeconds: 300},
-		{sizeMin: 9000, sizeMax: math.MaxInt32, ttlSeconds: 600},
+		{sizeMin: 1800, sizeMax: math.MaxInt32, ttlSeconds: 300},
```
Comment (wojtek-t) [inline pkg/controller/ttl/ttl_controller.go:111]: Can you add corresponding tests in:
https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/ttl/ttl_controller_test.go#L207
?

Comment (marseel) [reply pkg/controller/ttl/ttl_controller.go:111]: Done.

**Pull request merged**